GPU/GPUCommon.h

```diff
@@ -196,6 +196,21 @@ class GPUCommon : public GPUThreadEventQueue, public GPUDebugInterface {
 	GPUgstate GetGState() override;
 	void SetCmdValue(u32 op) override;
 
+	void UpdateUVScaleOffset() {
+#ifdef _M_SSE
+		__m128i values = _mm_slli_epi32(_mm_load_si128((const __m128i *)&gstate.texscaleu), 8);
```

Comment:
Hmm... this is at an offset of 0x48, no?  Why doesn't it need to be `_mm_loadu_si128`?

-[Unknown]